e2e/Microsoft.Azure.Devices.E2ETests/MethodE2ETests.cs

```diff
@@ -345,21 +345,15 @@ public void Cleanup()
             Tuple<string, string> deviceInfo = TestUtil.CreateDevice(DevicePrefix, hostName, registryManager);
             var assertResult = new TaskCompletionSource<Tuple<bool, bool>>();
             var deviceClient = DeviceClient.CreateFromConnectionString(deviceInfo.Item2, transport);
-
-// TODO: #193
-// DeviceClient.SetMethodHandler(string, MethodCallback, object)' is obsolete: 'Please use SetMethodHandlerAsync.
+            
 #pragma warning disable CS0618
-            deviceClient?.SetMethodHandler(MethodName,
```

Comment:
The intention for this test is to test the obsoleted SetMethodHandler() until we can remove it... so we will need to keep it

Reply:
+1 
#193 talks about removing the obsolete functions too. Do you want to replace these with the non-obsolete versions within the same commit or are you going to address these later?

Reply:
Just to clarify, should I be changing it back, or making two tests, one for new method and one for obsoleted one?

Reply:
Good point: since we didn't remove the API, we could add another test and remove this one when we actually remove the API (in v2.0).